.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ workflows:
               parameters:
                   php-version: ["8.0", "8.1"]
                   composer-dependencies: ["lowest", "highest"]
-      - test-on-windows
```

Comment:
Would we re-introduce this?

Reply:
Once we address the problem upstream, yes, I think we would want to re-enable this.